### DISCUSSION TITLE
❤️ Feature Request: Add DependsOn functionality to Trix field so you can conditionally show the field

### DISCUSSION THREAD

**Discussion by chrillep** [laravel/nova-issues#5425] [Ideas & Feature Requests] [answered]
want it

**Comment by crynobone** [comment]
Already exists.

**Reply by chrillep** [reply]
Sry for taking your time I must have misunderstood the docs and usage when I tried this.

**Reply by chrillep** [reply]
I still cant get it to work?
`"laravel/nova": "^4.22.2",`
<img width="495" alt="Skärmavbild 2023-03-25 kl  16 18 29" src="https://user-images.githubusercontent.com/1267931/227726139-68d8a9b3-9dd6-459e-a1a7-9a191fd27914.png">

**Comment by chrillep** [comment, marked as answer]
Fixad in https://nova.laravel.com/releases/4.23.0
